pkg/api/login_test.go

```diff
@@ -207,6 +207,48 @@ func TestLoginViewRedirect(t *testing.T) {
 			appURL:      "http://localhost:3000/",
 			status:      302,
 		},
+		{
+			desc:        "non grafana url without scheme",
```

Comment:
I know we are standardizing on Grafana and URL in the docs, so probably should do the same in the code:
```suggestion
			desc:        "non-Grafana URL without scheme",
```